pipelined/bevy_sprite2/src/render/mod.rs

```diff
@@ -400,40 +527,30 @@ impl Draw<Transparent2d> for DrawSprite {
         view: Entity,
         item: &Transparent2d,
     ) {
-        const INDICES: usize = 6;
         let (sprite_meta, image_bind_groups, pipelines, views, sprites) = self.params.get(world);
         let view_uniform = views.get(view).unwrap();
         let sprite_meta = sprite_meta.into_inner();
         let image_bind_groups = image_bind_groups.into_inner();
-        let extracted_sprite = sprites.get(item.entity).unwrap();
+        let sprite_batch = sprites.get(item.entity).unwrap();
         if let Some(pipeline) = pipelines.into_inner().get(item.pipeline) {
             pass.set_render_pipeline(pipeline);
-            pass.set_vertex_buffer(0, sprite_meta.vertices.buffer().unwrap().slice(..));
-            pass.set_index_buffer(
-                sprite_meta.indices.buffer().unwrap().slice(..),
-                0,
-                IndexFormat::Uint32,
-            );
```

Comment:
Why stop using an index buffer?

Reply:
The performance ended up being _slightly_ worse with an index buffer. Small enough to not really matter, but removing the index buffer produced simpler / smaller code so I opted for it.